vendor/github.com/prometheus/prometheus/tsdb/head.go

```diff
@@ -494,6 +494,10 @@ func newHeadMetrics(h *Head, r prometheus.Registerer) *headMetrics {
 				60 * 60 * 12, // 12h
 			},
 		}),
+		mmapChunksTotal: prometheus.NewCounter(prometheus.CounterOpts{
+			Name: "prometheus_tsdb_mmap_chunks_total",
```

Comment:
Do we want to expose this value in ingesters metrics and aggregate over all tenants?

Reply:
yeah, why not; updated in https://github.com/grafana/mimir/pull/5927/commits/314bb7279d1329dd1aed3c68a65889a54e236161